ehr2meds/preMEDS/utils.py

```
@@ -232,3 +217,39 @@ def unroll_columns(df: pd.DataFrame, concept_config: dict) -> List[pd.DataFrame]
+    value_cols = cfg.get("source_cols")
+    target_col = cfg.get("target_name")
+    prefix_col = cfg.get("prefix_col")
+    prefix_map = cfg.get("prefix_map")
+    id_cols = [c for c in df.columns if c not in value_cols]
+    df_melted = df.melt(id_vars=id_cols, value_vars=value_cols, var_name="source", value_name=target_col)
+
+    # Add prefix
+    df_melted[prefix_col] = df_melted["source"].map(prefix_map) + df_melted[prefix_col].astype(str)
+
+    # # Drop columns
+    cols_to_keep = list(set([prefix_col, target_col] + id_cols))
```

Ruff (0.15.10)

[warning] Consider `[prefix_col, target_col, *id_cols]` instead of concatenation

Replace with `[prefix_col, target_col, *id_cols]`

(RUF005)